config/application.rb

```diff
@@ -15,5 +15,12 @@ class Application < Rails::Application
       g.orm :active_record
       g.template_engine :haml
     end
+
+    config.before_configuration do
+      env_file = File.join(Rails.root, 'config', 'local_env.yml')
+      YAML.load(File.open(env_file)).each do |key, value|
+        ENV[key.to_s] = value
+      end if File.exists?(env_file)
```

Comment:
Nice, you can use `figaro` gem for this and it has a nice feature that updates your heroku server with the ENV vars.